frontend/src/views/ScheduleView.vue

```diff
@@ -117,6 +81,7 @@ onMounted(async () => {
   // Don't actually load anything during the FTUE
   if (route.name === 'setup') {
     // Setup a fake schedule so the schedule creation bar works correctly...
+    // TODO: move that to the schedule store as initial/default value
```

Comment:
@MelissaAutumn I think it would be more convenient to have the initial schedule object in the store instead of here. If you agree, I can move it.

Reply:
That sounds like a good idea!

Reply:
Done.